### PR TITLE
Chore: Upgrade axios to v0.22

### PR DESCRIPTION
**Pull request by mahboubii**
high severity security issue https://github.com/advisories/GHSA-cph5-m8f7-6c5x

### CODE REVIEW

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: src/errors.ts [hunk 1]
```diff
@@ -67,11 +67,11 @@ export class MissingSchemaError extends ErrorAbstract {}
  * @param  {object} data
  * @param  {object} response
  */
-export class StreamApiError extends ErrorAbstract {
+export class StreamApiError<T> extends ErrorAbstract {
```
Comment (arnautov-anton) [inline src/errors.ts:70]: No defaults?

**Comment by arnautov-anton** [comment]
:+1: :taco:

**Review by arnautov-anton — APPROVED** [review]

File: src/errors.ts (hunk 1 above)
Comment (mahboubii) [reply src/errors.ts:70]: will add later if needed, good for now

**Pull request merged**